packages/components/addon/components/hds/cta.js

```diff
@@ -0,0 +1,17 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class HdsCtaComponent extends Component {
+  @action
+  doRouteTransition(e) {
+    console.log('This is triggered by every instance of the CTA in the page');
+    if (document.activeElement) {
```

Comment:
btw @didoo this didn't work correctly because you were missing `hasFocus()`. If you change this line (line 8) to this: 
`if (document.activeElement.hasFocus()) {`
Then this code works. Just thought you would want to know!